modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -76,7 +76,7 @@ def build_appointment_model(appointment_hash)
           sta6aid = facility_id
           type = parse_by_appointment_type(appointment_hash, appointment_hash[:kind])
           start_date_utc = start_date_utc(appointment_hash)
-          time_zone = time_zone(facility_id)
+          time_zone = appointment_hash.dig(:location, :timezone, :timeZoneId)
```

Comment:
Will it ever be the case that the timezone or timeZoneId will be nil or not exist? Wondering if you need to account for the case where bad or empty data could exist.

Reply:
Dig will return nil if either of those is missing or nil. Nil values for time_zone were already possible, so I don't think this fundamentally change anything.

Reply:
Luckily, `dig` is pretty robust where if it can't any of the keys in the hash, it'll return a nil and won't throw an error. I could potentially fall back to the hardcoded timezones based on the facility id it was using before if it does return a nil.

Reply:
That might not be a bad idea.